docs/UserGuide.adoc

```diff
@@ -203,16 +203,16 @@ Format: `list [y/YEAR]`
 
 [NOTE]
 ====
-* If no modules have been added to a specified year or no modules have been added to ModulePlanner, will show an empty list.
```

Comment:
I think added is a better word to use since one can list all modules in year 4 even though one has not taken those modules in year 4

Reply:
Done